### ISSUE TITLE
cannot move focus between buttons on message bar with keyboard

### ISSUE THREAD

**Issue by chrisdias** [Microsoft/vscode#4396]
Related to #3769 

I was able to get focus to the message bar on Windows (didn't try OSX yet) but i can't move between the buttons. Tab moves me off the message bar, left and right arrow don't do anything.

![image](https://cloud.githubusercontent.com/assets/1487073/13865009/e91e06b0-ec63-11e5-8105-d3ca9b7f419d.png)

**Comment by bpasero** [comment]
![untitled](https://cloud.githubusercontent.com/assets/900690/13869808/d9df91da-ecd8-11e5-98b6-52cdda418e66.gif)

Works for me. I am on Mac though. Can you chrome inspect the 3 buttons all have a tabindex attribute?

**Comment by bpasero** [comment]
Same thing works for me on Windows too.

**Comment by chrisdias** [comment]
i see a tabstop="0" for all three buttons, but now i see what the problem is.  Shift+Tab moves me right to left and tab moves me left to right.  since focus landed on the left most button, pressing tab again jumped out of the message.  

Here's what I did.  I put focus in the explorer and started pressing Shift+Tab to get to the message bar.  Focus went to the left most button. Now, i think to move from left to right I should press Tab (not shift+tab) and when I do focus moves away from the message bar.  

I expect that Shift Tab to move me right to left through the buttons, just like it does in the explorer buttons (e.g. new folder, new file). Also, when focus reaches the message bar i would expect focus to be on the right most button.  So, when i press Shift tab it moves right to left through the three buttons and pressing tab would move me left to right.

**Comment by bpasero** [comment]
Ah so you see an issue with the ordering. That I can maybe fix.

**Comment by bpasero** [comment]
@chrisdias please verify across all OS.

**Comment by chrisdias** [comment]
looks good!
